BetterPasswordPlugin.inc.php

```diff
@@ -208,17 +236,62 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+	/**
+	 * Create a temporary file with the data of password blacklists.
+	 * @return string temporary filename string
+	 */
+         function handleTempFile() {
+            import('lib.pkp.classes.file.PrivateFileManager');
+            $fileMgr = new PrivateFileManager();
+            $siteDao = DAORegistry::getDAO('SiteDAO');
+            $site = $siteDao->getSite();
+            $minLengthPass = $site->getMinPasswordLength();
+            $tempFileDir = realpath($fileMgr->getBasePath()) . DIRECTORY_SEPARATOR . 'passTmp';
```

Comment:
Let's name this directory after the plugin:
$tempFileDir = realpath($fileMgr->getBasePath()) . DIRECTORY_SEPARATOR . 'betterPassword';